src/Nethermind/Nethermind.JsonRpc/Nethermind.JsonRpc.csproj

```diff
@@ -16,6 +16,7 @@
     <ProjectReference Include="..\Nethermind.Crypto\Nethermind.Crypto.csproj" />
     <ProjectReference Include="..\Nethermind.Evm\Nethermind.Evm.csproj" />
     <ProjectReference Include="..\Nethermind.Facade\Nethermind.Facade.csproj" />
+    <ProjectReference Include="..\Nethermind.JsonRpc.Connectors\Nethermind.JsonRpc.Clients.csproj" />
```

Comment:
A bit odd situation where prefix-named package depends on suffix-named package. Most often it is the other way around.

Reply:
Ended up removing the project again, since it was not needed any longer. Though having RPC clients in a project without many of the current dependencies is not a bad idea.